tests/src/test/scala/limits/ThrottleTests.scala

```diff
@@ -327,8 +327,12 @@ class NamespaceSpecificThrottleTests
   val oneProps = getAdditionalTestSubject("oneSubject")
   wskadmin.cli(Seq("limits", "set", oneProps.namespace, "--invocationsPerMinute", "1", "--firesPerMinute", "1"))
 
+  // Create a subject where the rate limits are set to 1 for testing sequences
+  val oneSequenceProps = getAdditionalTestSubject("oneSequenceSubject")
```

Comment:
do you need an additional test subject vs reusing one `oneProps` subject above?

Reply:
Yes or I have to wait 1 minute.

Reply:
Ah - sneaky. Thanks.

Reply:
fwiw you only need the invocations per minute for the test.

Reply:
perhaps a comment :)